tests/test_hooks.py

```diff
@@ -0,0 +1,274 @@
+# -*- coding: utf-8 -*-
+from bobtemplates import hooks
+from mrbob.bobexceptions import SkipQuestion
+from mrbob.bobexceptions import ValidationError
+from mrbob.configurator import Configurator
+from mrbob.configurator import Question
+
+import pytest
+
+
+def test_to_boolean():
+    # Initial simple test to show coverage in hooks.py.
+
+    # Positive Return Checks
+    assert hooks.to_boolean(None, None, True)
+    assert hooks.to_boolean(None, None, 'y')
+    assert hooks.to_boolean(None, None, 'yes')
+    assert hooks.to_boolean(None, None, 'true')
+    assert hooks.to_boolean(None, None, '1')
+
+    # Negative Return Checks
+    assert hooks.to_boolean(None, None, False) is False
+    assert hooks.to_boolean(None, None, 'n') is False
+    assert hooks.to_boolean(None, None, 'no') is False
+    assert hooks.to_boolean(None, None, 'false') is False
+    assert hooks.to_boolean(None, None, '0') is False
+
+    # Error Check
+    with pytest.raises(ValidationError, msg='Value must be a boolean (y/n)'):
+        assert hooks.to_boolean(None, None, 'spam')
+
+
+def test_validate_packagename():
+    with pytest.raises(AttributeError):
+        hooks.validate_packagename(None)
+
+    configurator = Configurator(
```

Comment:
no. that is like it should.

I do test various variants within one test method. 
As ``hooks.validate_packagename()`` should take a Configurator object but also tests against None did have for all of those case test steps. Maybe it would be good to put comments for each case.